man/ostree.repo-config.xml

```diff
@@ -404,6 +404,17 @@ License along with this library. If not, see <https://www.gnu.org/licenses/>.
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>bls-append-except-default</varname></term>
+        <listitem><para>A semicolon seperated list of key-value pairs. For example:
+        <literal>bls-append=key1,value1;key2,value2</literal>. These key-value pairs
```

Comment:
`GKeyFile`s support string lists natively. It might be simpler to document the key as a string list instead (which actually uses the same `;`-separated format documented here) and in the code we use [`get_string_list`](https://docs.gtk.org/glib/method.KeyFile.get_string_list.html) instead of parsing the string ourselves.